mllib/src/main/scala/org/apache/spark/ml/classification/GBTClassifier.scala

```diff
@@ -240,6 +240,17 @@ final class GBTClassificationModel private[ml](
     s"GBTClassificationModel (uid=$uid) with $numTrees trees"
   }
 
+  /**
+   * Estimate of the importance of each feature.
+   *
+   * Each feature's importance is the average of its importance across all trees in
+   * the ensemble.
```

Comment:
Here too, I'd like to note that feature importances are calculated in the same way as in scikit-learn and the Friedman paper, and that they sum to 1.